sdcm/provision/common/utils.py

```diff
@@ -86,74 +89,86 @@ def configure_syslogng_target_script(host: str, port: int, throttle_per_second:
         """.format(host=host, port=port, hostname=hostname, throttle_per_second=throttle_per_second))
 
 
-def configure_rsyslog_set_hostname_script(host: str) -> str:
+def configure_rsyslog_set_hostname_script(hostname: str) -> str:
```

Comment:
Such host -> hostname renaming looks like a separate commit. For now it only complicates already big commit.

Reply:
done